CONTRIBUTING.adoc

```diff
@@ -480,11 +480,11 @@ link:https://github.com/jenkins-infra/jenkins.io/new/master/content/_data/events
 Name the file using the pattern `<DATE>-<CITY><OPTIONAL_ID>.adoc`:
 
 * DATE -
-  The is the date of the event written as `YYYY-MM-DD`.
+  The date of the event written as `YYYY-MM-DD`.
   For a multi-day event, use the starting day.
 * CITY -
-  The name of the city in lowercase letters without modifiers/accents
-  (only the characters "a - z") and using dashes instead of spaces.
+  The name of the city in lower-case letters without modifiers/accents
```

Comment:
I don't have a strong opinion one way or the other, but I like the description at

https://english.stackexchange.com/questions/59409/lowercase-lower-case-or-lower-case

They say

> Warren Chappell, Robert Bringhurst, and the printed Unicode Standard all use lowercase exclusively.

I haven't looked to see how it is used elsewhere in the documentation, but I'd prefer to use the same word as the Unicode Standard, if only because there is lots of Unicode in Java.

```suggestion
  The name of the city in lowercase letters without modifiers/accents
```

Reply:
Yes, I agree. "lowercase" is better. There are, I think, three other places in this document with "lower-case" so I will make a PR on those later.